src/main/menus/tray.js

```diff
@@ -29,6 +29,14 @@ function createTemplate(mainWindow, config) {
     }), {
       type: 'separator'
     }, {
+      label: 'Settings',
+      click: () => {
+        mainWindow.loadURL('file://' + __dirname + '/browser/settings.html');
+        mainWindow.show();
```

Comment:
I've changed it. Btw, i just ran a search regarding show or restore. It is used in a lot of place on the code. Could it be good to call one method to do it or add the equivalent directly onto the mainWindow object ?